testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/MyResourcesTest.java

```diff
@@ -30,18 +30,20 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 import static java.util.Collections.singletonList;
 import static org.hamcrest.core.StringEndsWith.endsWith;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 import static org.keycloak.representations.idm.CredentialRepresentation.PASSWORD;
 import static org.keycloak.testsuite.auth.page.AuthRealm.TEST;
 import static org.keycloak.testsuite.util.WaitUtils.pause;
 import static org.keycloak.testsuite.util.WaitUtils.waitForModalFadeIn;
 import static org.keycloak.testsuite.util.WaitUtils.waitForModalFadeOut;
 
-public class MyResourcesTest extends AbstractAccountTest {
+public class MyResourcesTest extends BaseAccountPageTest {
```

Comment:
Please add a test for the Pending requests dialog and Edit resource dialog to check if it displays all the users it is supposed to.